src/constants/initialImages.ts

```diff
@@ -0,0 +1,8 @@
+export const initialImages = [
```

Comment:
I'd suggest to download images and keep them as assets. Then you don't need to worry if url changes some day in the future or if the image is simply deleted